mirage/views/draft-registration.ts

```diff
@@ -0,0 +1,22 @@
+import { HandlerContext, Schema } from 'ember-cli-mirage';
+
+export function createDraftRegistration(this: HandlerContext, schema: Schema) {
+    const attrs = this.normalizedRequestAttrs('draft-registration');
+    const now = (new Date()).toISOString();
+    const userId = schema.roots.first().currentUserId;
+    let currentUser;
+
+    if (userId) {
+        currentUser = schema.users.find(userId);
+    }
+
+    const draftRegistration = schema.draftRegistrations.create({
```

Comment:
Should also put `branchedFrom` in here if we want it to hit it. (Mirage doesn't use the Factory, so anything that we want to include should be in this file instead)

Reply:
`branchedFrom` should be in `attrs`